src/Symfony/Bridge/Doctrine/Security/MagicLink/MagicLoginLinkTokenEntityTrait.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Security\MagicLink;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Trait that can be used by your entity to help implement StoredMagicLinkTokenInterface.
+ *
+ * @author Ryan Weaver <ryan@symfonycasts.com>
+ */
+trait MagicLoginLinkTokenEntityTrait
+{
+    /**
+     * @ORM\Column(type="string", length=20, unique=true)
+     */
+    private $selector;
+
+    /**
+     * @ORM\Column(type="string", length=100)
+     */
+    private $hashedVerifier;
+
+    /**
+     * @ORM\Column(type="datetime_immutable")
+     */
+    private $createdAt;
```

Comment:
Just asking: do we really need to store `createdAt` or would it be enough to store `expiresAtTimestamp` ?

Reply:
We totally do NOT need it... and yea... the user could add it if they want, so I'll remove it